### ISSUE TITLE
Communication with two cameras?

### ISSUE THREAD

**Issue by tenisak** [reflex33/VISCA-Control#2]
Thanks a lot for a great library! 
Is it possible to communicate with two cameras at once? I have two cameras EVI-D70 connected via RS232 as shown in the manual (page 31 [here](https://pro.sony.com/bbsccms/assets/files/mkt/remotemonitoring/manuals/rm-EVID70_D70P_Technical_Manual.pdf)): 
![image](https://user-images.githubusercontent.com/34473563/36476572-9ce8bcbe-16fe-11e8-9116-b236bd257270.png)

**Comment by reflex33** [comment]
The current code assumes a single camera. I don't believe there is anything that is stopping that from being supported, but would it would require some work support. I don't have a pair (or more) of cameras to do the coding/testing though. My main concern would be the timing of messages and if there are any extra messages coming from the additional cameras. The visca protocol is a bit hard to follow and has many special cases where commands behave differently than expected.

**Comment by tenisak** [comment]
Thank you very much for your reply. My idea is to change the `cameraNum` externally with a new method (eg. `SetCameraNum(num)`). Do you think that would be enough? Or would it be necessary to make even more major changes?

**Comment by reflex33** [comment]
At a minimum the connect function will need to be modified, as it assume's a single camera for now. It look for serial communication back from the camera and the camera number it receives is used as the camera number internally. So setting it externally won't be enough.

The next major issue is the serial port. Multiple cameras will still communicate through a single port. You issue a command to the camera (e.g. camera. JogPanTiltDegrees(d)), which will look at its internal camera number and send the command over the serial port. Having a second camera instance with a different camera number will not be good here as both will need access to the same serial port.

More work would have to be done make it so the camera class can handle more than one camera (maybe we don't call it camera anymore at that point). Then you would communicate to all of the visca cameras through one class instance.
